src/tests/functional/plugin/shared/src/execution_graph_tests/runtime_precision.cpp

```diff
@@ -123,7 +123,23 @@ TEST_P(ExecGraphRuntimePrecision, CheckRuntimePrecision) {
         if (rtIter == rtInfo.end())
             FAIL() << "Runtime precision is not found for node: " << opIter->get()->get_friendly_name();
 
-        if (expectedPrc.second.name() != rtIter->second.as<std::string>())
+        bool success_status = false;
+        success_status = std::regex_match(expectedPrc.second.name(),
```

Comment:
Solution looks overcomplicated. Can it be simplified by using some converters like `convertPrecision` etc.?